website/content/docs/nia/compatibility.mdx

```diff
@@ -7,28 +7,26 @@ description: >-
 
 # Compatibility
 
-## Consul
-
-Below are the supported Consul versions with compatible Consul-Terraform-Sync versions. The latest Consul-Terraform-Sync binary targets supporting the latest patch version of the three most recent Consul minor versions.
-
-| Consul Version               | Compatible Consul-Terraform-Sync Version |
-| ---------------------------- | ---------------------------------------- |
-| 1.8+                         | 0.1+                                     |
-
-## Terraform
-
-Consul-Terraform-Sync is compatible with the following Terraform OSS versions:
-
-| Consul-Terraform-Sync | Compatible Terraform Version |
-| --------------------- | ---------------------------- |
-| 0.2+                  | 0.13 - 1.0                   |
-| 0.1                   | 0.13 - 0.14                  |
-
-## Terraform Cloud
-
-Consul-Terraform-Sync integration with Terraform Cloud is supported for the following:
-
-| Consul-Terraform-Sync Enterprise | Terraform Cloud            | Version            |
-| -------------------------------- | -------------------------- | ------------------ |
-| 0.4+                             | Terraform Cloud            | Latest             |
-| 0.3+                             | Terraform Enterprise       | v202010-2 - Latest |
+## Consul-Terraform-Sync
+
+Below are the supported Consul, Terraform, and Vault versions with compatible Consul-Terraform-Sync versions.
```

Comment:
```suggestion
The following table describes Consul-Terraform-Sync version compatibility for Consul, Terraform, and Vault.
```
Just trying to see if I can make this cleaner. Ignore if it doesn't seem like an improvement